src/products/entities/product.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Base } from 'src/shared/entities/base.entity';
+import { Column, Entity } from 'typeorm';
+
+@Entity('products')
+export class ProductEntity extends Base {
+  @Column({ type: 'varchar', unique: true })
+  name: string;
+
+  @Column({ type: 'text' })
```

Comment:
Can you add length and if is nulleable fields?

tip: You can use `typeorm-model-generator` library to generate the entity file from your database with all information